test/10_cpp_standards/cibuildwheel_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys 
+
+import pytest
+
+import utils
+
+
+def test_cpp11(tmp_path):
```

Comment:
Why do we need this `tmp_path`?

Reply:
to store result of different cibuildwheel call in different directories. It allows check if result is correct. Before this only first test check content  of wheelhouse directory, so there is no collision.

Reply:
But we don't need this in other tests. See for example `02_...`.

Should we then adapt the other tests, or is this change you made not really necessary because `pip wheel` will already build in a separate directory)?

Reply:
I see that package name from `01_test` and `02_test` use same name and same version of package. 
When I run test on my personal computer I see that, when I use different names I got wheels for all names (you may see that i name projects `spam11`, `spam14` and `spam17`). 

So it looks like `02_test` may pass if it silently fail, because `01_test` already produce wheels in `wheelhouse`. 

I think that `cibuildwheel` should use `tmp_path` fixture or clean `wheelhouse` before test

Reply:
OK, yes, so we should make all the tests follow the same structure! But maybe that's a different PR, then.

Reply:
I'm working on adding a test for `manylinux2014` and I ran into the same issue. I'll try to make a PR that reworks a bit testing that should answer @Czaki remarks.

Reply:
I just finishing refactor of test. https://github.com/Czaki/cibuildwheel/tree/test_fix

Reply:
So is this now OK, or do we need to still fix this to work with the refactoring of the tests we did some weeks ago?